bootstrap/bootstrapper.debug.yaml

```diff
@@ -0,0 +1,55 @@
+---
```

Comment:
Add a comment explaining what this manifest is for please.

Reply:
done